README.md

```diff
@@ -21,53 +22,29 @@ Runtime:
 - Python 2.7
 - Z3 (for the Manticore backend)
 
-## Building
-
-DeepState is a static library, used to write test harnesses, accompanied by command-line _executors_ written in Python. Below we describe how to build the library and accompanying Python package.
-
-### Ubuntu 16.04 (Xenial)
-
-Suppose the DeepState project source resides in the directory `$DEEPSTATE`.
-
-First, install the build dependencies:
-
-```shell
-$ sudo apt update
-$ sudo apt install build-essential gcc-multilib cmake python python-setuptools
-```
-
-Set up a build directory and `cd` into it:
-
-```shell
-$ mkdir $DEEPSTATE/build
-$ cd $DEEPSTATE/build
-```
-
-From the build directory, generate Makefiles using CMake:
-
-```shell
-$ cmake $DEEPSTATE
-```
-
-Finally, build the library and package:
+## Building on Ubuntu 16.04 (Xenial)
 
 ```shell
+$ sudo apt update && sudo apt-get install build-essential gcc-multilib cmake python python-setuptools
+$ git clone https://github.com/trailofbits/deepstate deepstate
+$ mkdir deepstate/build && cd deepstate/build
+$ cmake ../
 $ make
 ```
 
-## Usage
+## Installing
 
-After building, you can use DeepState by installing the resulting Python package, e.g. into a virtualenv. For example, from some working directory, with the `virtualenv` tool installed:
+Install the DeepState Python package. For example, assuming the DeepState build resides in `$DEEPSTATE`:
```

Comment:
"Install the DeepState Python package" is maybe redundant here given the section head